src/main/java/com/easypost/service/EasyPostClient.java

```diff
@@ -145,6 +156,38 @@ public EasyPostClient(String apiKey, int connectTimeoutMilliseconds, int readTim
         this.webhook = new WebhookService(this);
     }
 
+    /**
+     * Subscribes to a request hook from the given function.
+     * @param function
+     */
+    public void subscribeToRequestHook(Function<HashMap<String, Object>, Object> function) {
```

Comment:
I feel like there was some discussion with @Justintime50 RE: whether we want to support triggering multiple callbacks in our library, or simply one pre-request and one post-request callback, which users can trigger additional callbacks from there.

Reply:
In Python CL, which has been merged, is one callback in our lib

Reply:
So we don't need to subscribe to multiple callbacks?

Reply:
Not 100% sure what you mean by `multiple callbacks`